modules/consul/metadata.yaml

```diff
@@ -1,120 +1,161 @@
-name: consul
-title: Consul collector
+meta:
+  plugin_name: go.d.plugin
+  module_name: consul
+  alternative_monitored_instances: []
+  monitored_instance:
+    name: Consul
+    link: "https://www.consul.io/"
+    categories:
+      - data-collection.service-discovery-registry
+    icon_filename: "consul.png"
+  related_resources:
+    integrations:
+      list: []
+  info_provided_to_referring_integrations:
+    description: ""
+  keywords:
+    - service networking platform
+    - hashicorp
```

Comment:
Do we need to add the app name as the keyword (add `consul` here)?

Reply:
I'd say no, probably it will be searchable by its name, maybe @kapantzak can confirm that we don't need consul as keyword because that is the name